libraroo-web/src/bookCatalog/components/CreateBookCatalog.tsx

```diff
@@ -0,0 +1,97 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import BookCatalogService from "../services/bookCatalog.service";
+import { Alert, Box, Button, Container, TextField } from "@mui/material";
+
+const CreateBookCatalog: React.FC = () => {
+  const [author, setAuthor] = useState("");
+  const [title, setTitle] = useState("");
+  const [language, setLanguage] = useState("");
+  const [error, setError] = useState<string | null>(null);
+  const [loading, setLoading] = useState(false);
+
+  const navigate = useNavigate(); // React Router's navigation hook
+
+  const handleSubmit = async (event: React.FormEvent) => {
+    event.preventDefault();
+    setLoading(true);
+    setError(null); // Clear previous errors
+
+    try {
+      await BookCatalogService.addBookCatalog({
+        author,
+        title,
+        language,
+      });
+
+      navigate("/");
+    } catch (err: any) {
+      setError("Invalid email or password");
+    } finally {
+      setLoading(false);
+    }
+  };
```

Comment:
wtf

Reply:
oopsooshhhhhhhhh

Reply:
I dont think this should navigate you to the home screen, maybe the book catalog list screen, add a TODO comment